staging/src/k8s.io/apiserver/pkg/server/config.go

```diff
@@ -583,6 +584,18 @@ func installAPI(s *GenericAPIServer, c *Config) {
 			routes.DefaultMetrics{}.Install(s.Handler.NonGoRestfulMux)
 		}
 	}
+
+	// so far, only logging related endpoints are considered valid to add for these debug flags.
+	routes.DebugFlags{}.Install(s.Handler.NonGoRestfulMux, "v", routes.StringFlagPutHandler(
+		routes.StringFlagSetterFunc(func(val string) (string, error) {
+			var level glog.Level
+			if err := level.Set(val); err != nil {
```

Comment:
Does this actually change the global verbosity level? I don't understand how messing with a local variable could do that?

Reply:
Yes. I believe this can't work unless you affect this variable somehow: https://github.com/golang/glog/blob/master/glog.go#L465

Reply:
You're right, the level.Set method modifies a global variable (??????)